api/src/main/java/uk/gov/dhsc/htbhf/claimant/model/VoucherEntitlementDTO.java

```diff
@@ -12,21 +13,27 @@
 public class VoucherEntitlementDTO {
 
     @JsonProperty("vouchersForChildrenUnderOne")
+    @ApiModelProperty(notes = "The number of vouchers for children under one year old", example = "2")
     private int vouchersForChildrenUnderOne;
 
     @JsonProperty("vouchersForChildrenBetweenOneAndFour")
+    @ApiModelProperty(notes = "The number of vouchers for children between one and 4 years old", example = "1")
     private int vouchersForChildrenBetweenOneAndFour;
 
     @JsonProperty("vouchersForPregnancy")
+    @ApiModelProperty(notes = "The number of vouchers for pregnancy", example = "1")
     private int vouchersForPregnancy;
 
     @JsonProperty("totalVoucherEntitlement")
+    @ApiModelProperty(notes = "The total number of vouchers the claimant is entitled to", example = "4")
     private int totalVoucherEntitlement;
 
     @JsonProperty("voucherValueInPence")
+    @ApiModelProperty(notes = "The financial value of a single voucher", example = "310")
     private int voucherValueInPence;
```

Comment:
Note that I have not changed the name used in our API - that remains voucherValueInPence, though I have added notes explaining it.

Reply:
That seems an odd choice, any reason why? I think calling it `singleVoucherValueInPence` would be helpful and consistent.

Reply:
I've added singleVoucherValueInPence - we'll need to update the front-end then we can remove voucherValueInPence.

Reply:
Turns out it wasn't used in the front-end, so we've been able to rename it in this PR after all.